PennyMe/ViewController.swift

```diff
@@ -469,6 +510,17 @@ class ViewController: UIViewController, UITextFieldDelegate, UIGestureRecognizer
                         // re-add annotations here
                         self?.isVisible = [:]
                         self?.addAnnotationsIteratively()
+                        
+                        // count how many machines we have in total
+                        totalMachines = 0
+                        machinesByArea = [:]
+                        for machine in self!.artworks {
+                            if machine.machineStatus != "retired" {
```

Comment:
This means that we count "out-of-order" machines. Fine for me

Reply:
Yes good point, that was intended